### DISCUSSION TITLE
MortgageCalculator/CurrencyCalculator #3

### DISCUSSION THREAD

**Discussion by JessicaOrj** [JessicaOrj/Tafe240267#1] [General]
This discussion is to talk about the individual code, let me know if you have any question or need a hand with it.
Do you have any question related to your Mortgage calculator ?

**Comment by JessicaOrj** [comment]
Hello, both Mortgage_Calculator and Currency_Calculator are done.
Are we ready to merge now ?

**Comment by ghost** [comment]
Yeah, I did the final test for both parts of the code, and everything seems ok. There were no any conflicts.

**Comment by JessicaOrj** [comment]
ok Haunly, merge to de development branch is done now, Again there were no conflicts. We can move on to the next part of the assessment.
